pkg/api/types.go

```diff
@@ -1205,6 +1208,21 @@ type KeyToPath struct {
 	Mode *int32
 }
 
+// LocalVolume represents a local volume accessible only from one node
+type LocalVolumeSource struct {
+	// NodeName represents the node that the volume is located at
+	NodeName string
```

Comment:
Since they are the same (label is populated based on NodeName field), why can't we use NodeName directly?  I haven't looked at scheduler part, but to answer my own question, I guess this is to confirm to the same logic applied to affinity/anti-affinity, and can also benefit from code reuse?